modules/exploits/multi/http/spring_framework_rce_spring4shell.rb

```diff
@@ -268,11 +264,35 @@ def exploit
     modify_class_loader(properties)
 
     check_log_file
-    
+
     handler
-    
   end
 
-end
+  # Retry the block until it returns a truthy value. Each iteration attempt will
+  # be performed with expoential backoff. If the timeout period surpasses, false is returned.
+  def retry_until_true(timeout:)
```

Comment:
Copied this over from another module in the framework. Planning on putting it into a library very soon.